docs/website/docs/dlt-ecosystem/verified-sources/openapi-generator.md

```diff
@@ -0,0 +1,198 @@
+├── .gitignore
+├── pokemon_pipeline.py # your pipeline file that you can execute
+├── README.md           # a list of your endpoints with some additional info
+└── requirements.txt    # the pip requirements for your pipeline
+```
+
+> If you re-generate your pipeline, you will be prompted to continue if this folder exists. If you select yes, all generated files will be overwritten. All other files you may have created will remain in this folder.
+
+## A closer look at your rest_api dictionary in pokemon/__init__.py
+
+This file contains the configuration dictionary for the [dlt rest_api](https://dlthub.com/docs/dlt-ecosystem/verified-sources/rest_api) source which is the main result of running this generator. For our Pokemon example, we have used an OpenAPI 3 spec that works out of the box. The result of this dictionary depends on the quality of the spec you are using, whether the API you are querying actually adheres to this spec, and whether our heuristics manage to find the right values. You can edit this file to adapt the behavior of the dlt rest_api accordingly. Please read our [dlt rest_api](https://dlthub.com/docs/dlt-ecosystem/verified-sources/rest_api) docs to learn how to do this and play with our detailed [Google Colab example](https://colab.research.google.com/drive/1MRZvguOTZj1MlkEGzjiso8lQ_wr1MJRI?usp=sharing#scrollTo=LHGxzf1Ev_yr).
+
+The generated dictionary will look something like this:
+
+```py
+{
+    "client": {
+        "base_url": base_url,
+        # -> the detected common paginator
+        "paginator": {
+            ...
+        },
+    },
+    # -> your two endpoints
+    "resources": [
+        {
+            # -> A primary key could not be inferred from
+            # the spec; usual suspects such as id, pokemon_id, etc.
+            # are not defined. You can add one if you know.
+            "name": "pokemon_list",
+            "table_name": "pokemon",
+            "endpoint": {
+                # -> the results seem to be nested in { results: [...] }
+                "data_selector": "results",
+                "path": "/api/v2/pokemon/",
+            },
+        },
+        {
+            "name": "pokemon_read",
+            "table_name": "pokemon",
+            # -> A primary key *name* is assumed, as it is found in the 
+            # url.
+            "primary_key": "name",
+            "write_disposition": "merge",
+            "endpoint": {
+                "data_selector": "$",
+                "path": "/api/v2/pokemon/{name}/",
+                "params": {
+                    # -> your detected transformer settings
+                    # this is a child endpoint of the pokemon_list
+                    "name": {
+                        "type": "resolve",
+                        "resource": "pokemon_list",
+                        "field": "name",
+                    },
+                },
+            },
+        },
+    ],
+}
+```
+
```

Comment:
```suggestion
You can edit this file to adapt the behavior of the dlt rest_api accordingly. Read our [dlt rest_api](./rest_api) docs to learn how to configure rest_api source and play with our detailed [Google Colab example](https://colab.research.google.com/drive/1MRZvguOTZj1MlkEGzjiso8lQ_wr1MJRI?usp=sharing#scrollTo=LHGxzf1Ev_yr).
```

Possible suggestion: to move the text + link to the config from the intro text to outro to remind the reader again what's that about and link for more.

Reply:
@burnash I'm not quite sure what you mean?

Reply:
my idea is to place this text:

> You can edit this file to adapt the behavior of the dlt rest_api accordingly. Read our [dlt rest_api](./rest_api) docs to learn how to configure rest_api source and play with our detailed [Google Colab example](https://colab.research.google.com/drive/1MRZvguOTZj1MlkEGzjiso8lQ_wr1MJRI?usp=sharing#scrollTo=LHGxzf1Ev_yr).

right under the config example:
![Screenshot 2024-05-27 at 17 20 20](https://github.com/dlt-hub/dlt/assets/264674/3a54a5a4-bae8-42c8-be61-701388c4c1b0)